include/seqan3/range/views/minimiser.hpp

```diff
@@ -542,8 +539,26 @@ namespace seqan3::views
  * \details
  *
  * A minimiser is the smallest value in a window. For example for the following list of hash values
- * [28, 100, 9, 23, 4, 1, 72, 37, 8] and 4 as `window_size`, the minimiser values are [9,4,1].
+ * `[28, 100, 9, 23, 4, 1, 72, 37, 8]` and 4 as `window_size`, the minimiser values are `[9, 4, 1]`.
  *
+ * The minimiser can be calculated for one given range or for two given ranges, where the minimizer is the smallest
+ * value in both windows. For example for the following list of hash values `[28, 100, 9, 23, 4, 1, 72, 37, 8]` and
+ * `[30, 2, 11, 101, 199, 73, 34, 900]` and 4 as `window_size`, the minimiser values are `[2, 4, 1]`.
+ *
+ * Note that in the interface with the second underlying range the const-iterable property will only be preserved if
+ * both underlying ranges are const-iterable.
+ *
+ * ### Robust winnowing
+ *
+ * In case there are multiple minimal values within one window, the minimum and therefore the minimizer is ambiguous.
+ * We choose the right most value as the minimizer of the window, and when shifting the window, the minimizer is only
+ * changed if there appears a value that is strictly smaller as the current minimum. This approach is termed
+ * *robust winnowing* by [Chirag et al.](https://www.biorxiv.org/content/10.1101/2020.02.11.943241v1.full.pdf)
+ * and is proven to work especially well on repeat regions.
```

Comment:
```suggestion
 * ### Robust Winnowing
 *
 * In case there are multiple minimal values within one window, the minimum and therefore the minimiser is ambiguous.
 * We choose the rightmost value as the minimiser of the window, and when shifting the window, the minimiser is only
 * changed if there appears a value that is strictly smaller than the current minimum. This approach is termed
 * *robust winnowing* by [Chirag et al.](https://www.biorxiv.org/content/10.1101/2020.02.11.943241v1.full.pdf)
 * and is proven to work especially well on repeat regions.
```